src/OutputReaders/field_dataset.jl

```diff
@@ -80,3 +83,114 @@ function Base.show(io::IO, fds::FieldDataset)
 
     return print(io, s)
 end
+
+"""
+    FieldDataset(grid, field_names, times;
```

Comment:
```suggestion
    FieldDataset(grid, times, field_names;
```

this feels more natural to me, what do you think?